articles/references/templates_decorators/toc.yml

```diff
@@ -0,0 +1,6 @@
+- name: Template & Decorator Registrations
+  href: registrations.md
+- name: The Template Class
+  href: the_template_class.md
+- name: Decorator
+  href: decorator.md
```

Comment:
For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.